pkgs/tools/filesystems/e2fsprogs/default.nix

```diff
@@ -36,7 +36,7 @@ stdenv.mkDerivation rec {
 
   configureFlags =
     if stdenv.isLinux then [
-      "--enable-elf-shlibs"
+      (if shared then "--enable-elf-shlibs" else "--disable-elf-shlibs")
```

Comment:
Can you indicate with a comment whether `e2fsprogs` is one of the (few) problems where one can only build static XOR dynamic?

Reply:
I added a comment.